glass/points.py

```diff
@@ -52,7 +52,7 @@ def effective_bias(
     z: npt.NDArray[np.float64],
     bz: npt.NDArray[np.float64],
     w: RadialWindow,
-) -> npt.NDArray[np.float64]:
+) -> float | npt.NDArray[np.double]:
```

Comment:
- `np.trapezoid` can return a `float`
- Division between `np.float64` and `np.float64` can result in `np.double` ("float-any") for some reason.